migrations/versions/0082_set_international.py

```diff
@@ -0,0 +1,45 @@
+"""empty message
+
+Revision ID: 0082_set_international
+Revises: 0081_noti_status_as_enum
+Create Date: 2017-05-05 15:26:34.621670
+
+"""
+from datetime import datetime
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = '0082_set_international'
+down_revision = '0081_noti_status_as_enum'
+
+
+def upgrade():
+    conn = op.get_bind()
+    start = datetime.utcnow()
+    notification_history = "select id from notification_history where international is null limit 10000"
+
+    results = conn.execute(notification_history)
+    res = results.fetchall()
+
+    while len(res) > 0:
+        conn.execute("update notification_history set international = False where id in ({})".format(
```

Comment:
Question. Is it worth having some sort of sleep in here to pause between each 10,000 rows?